app/graphql/directives.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable class-methods-use-this */
+const { gql } = require('apollo-server');
+const { SchemaDirectiveVisitor } = require('graphql-tools');
+const { defaultFieldResolver } = require('graphql');
+const getValidationFn = require('../utils/validationFunctions');
+const errors = require('../errors');
+const logger = require('../logger');
```

Comment:
'const' can be explicit in the first variable. Then separate the others with comas.
const a = foo,
 b = foo;